CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## [Unreleased]
 
+## [0.4.8] - 2022-01-10
+
```

Comment:
Missing info in CHANGELOG for this release

Reply:
Yea, that's intentional, as the included fix is not even worthy to mention :-)